include/odp/api/spec/timer.h

```diff
@@ -185,6 +185,27 @@ uint64_t odp_timer_ns_to_tick(odp_timer_pool_t timer_pool, uint64_t ns);
  */
 uint64_t odp_timer_current_tick(odp_timer_pool_t timer_pool);
 
+/**
+ * Sample tick value of timer pools
+ *
+ * Reads timer pool tick value simultaneously (or closely back-to-back) from all requested timer
+ * pools, and outputs those on success. Optionally, outputs corresponding source clock (HW) counter
```

Comment:
Should it be spelled out that the output data is written in the same order as timer pools.